pkg/drivers/kic/oci/network_create.go

```diff
@@ -100,20 +97,25 @@ func tryCreateDockerNetwork(subnetAddr string, subnetMask int, mtu int, name str
 		"--driver=bridge",
 		fmt.Sprintf("--subnet=%s", fmt.Sprintf("%s/%d", subnetAddr, subnetMask)),
 		fmt.Sprintf("--gateway=%s", gateway),
-		// options documentation https://docs.docker.com/engine/reference/commandline/network_create/#bridge-driver-options
-		"-o", "--ip-masq",
-		"-o", "--icc",
-		fmt.Sprintf("--label=%s=%s", CreatedByLabelKey, "true"),
-		name,
 	}
-
-	// adding MTU option because #9528
-	if mtu > 0 {
+	if ociBin == Docker {
+		// options documentation https://docs.docker.com/engine/reference/commandline/network_create/#bridge-driver-options
+		args = append(args, "-o")
+		args = append(args, "--ip-masq")
 		args = append(args, "-o")
-		args = append(args, fmt.Sprintf("com.docker.network.driver.mtu=%d", mtu))
+		args = append(args, "--icc")
+
+		// adding MTU option because #9528
+		if mtu > 0 {
```

Comment:
could podman network benefit from specifying the MTU ? (this fixed the issue of minikube in cloud shell not being able to pull large images like `golang:1.15` due to different cloud shell MTUs

Reply:
It might, but it will not benefit from "com.docker.network.driver.mtu" at any rate.

Reply:
It will be available in the future (like podman 3.0 or so), with a similar option

Reply:
maybe add a comment, to remember TODO add mtu for podman 3 (with an issue liunk)